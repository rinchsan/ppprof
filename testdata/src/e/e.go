package e

import (
	"fmt"
	"log"
	"net/http"
	"runtime"
)

func main() {
	runtime.SetBlockProfileRate(1)
	runtime.SetMutexProfileFraction(1)
	go func() {
		log.Fatal(http.ListenAndServe("localhost:6060", nil))
	}()

	fmt.Println("ppprof")
}
