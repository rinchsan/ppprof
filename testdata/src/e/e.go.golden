package e

import _ "net/http/pprof" // want "should import net/http/pprof"

import (
	"fmt"
	"log"
	"net/http"
	"runtime"
)

func main() {
	runtime.SetBlockProfileRate(1)
	runtime.SetMutexProfileFraction(1)
	go func() {
		log.Fatal(http.ListenAndServe("localhost:6060", nil))
	}()

	fmt.Println("ppprof")
}
